rest_framework_gis/filters.py

```diff
@@ -232,3 +290,21 @@ def filter_queryset(self, request, queryset, view):
             return queryset.order_by(-GeometryDistance(filter_field, point))
         else:
             return queryset.order_by(GeometryDistance(filter_field, point))
+
+    def get_schema_operation_parameters(self, view):
+        params = super().get_schema_operation_parameters(view)
+        params.append(
+            {
+                "name": self.order_param,
+                "required": False,
+                "in": "query",
+                "description": "",
+                "schema": {
+                    "type": "enum",
+                    "items": {"type": "string", "enum": ["asc", "desc"]},
+                    "example": "desc",
+                },
+                "style": "form",
+                "explode": False,
+            }
+        )
```

Comment:
This needs to return the params. It is throwing an error in drf spectacular

Reply:
this implementation is not based on DRF spectacular btw

Reply:
@auvipy yes but check all implementations of get_schema_operation_parameters, they all return data. This is the only one that does not. Settings params inside of get_schema_operation_parameters and then not returning it also does not make sense since no one can access these parameters, since they are not set on self or returned. It is a get function, which should return _something_. In this case the params.

Reply:
are you up for making a contribution? I will review it